web.py

```diff
@@ -348,11 +369,18 @@ def runloop():
             http_server = WSGIServer((self.addr, self.port), app)
             http_server.serve_forever()
 
+        def process_proposals_loop():
+            while True:
+                gevent.spawn(process_proposals)
+                gevent.sleep(2)
+
         def start_greenlets():
             logger.info("checking wallet...")
             self.check_wallet()
             logger.info("starting WebGreenlet runloop...")
             self.runloop_greenlet.start()
+            self.process_proposals_greenlet = gevent.Greenlet(process_proposals_loop)
+            self.process_proposals_greenlet.start()
```

Comment:
you also need to add `self.process_proposals_greenlet.kill()` in the `stop()` function

Reply:
created the greenlet.kill() for the new greenlet.